src/Ztm.Threading/TimerSchedulers/ThreadPoolScheduler.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Threading;
+
+namespace Ztm.Threading.TimerSchedulers
+{
+    public sealed class ThreadPoolScheduler : ITimerScheduler
+    {
+        public static readonly TimeSpan MaxDuration = TimeSpan.FromMilliseconds(4294967294);
```

Comment:
Why we use this value? I think we could use `TimeSpan.MaxValue` instead.

Reply:
Added document.